table/sst_file_writer.cc

```diff
@@ -161,6 +170,11 @@ struct SstFileWriter::Rep {
     auto ikey_and_end_key = tombstone.Serialize();
     builder->Add(ikey_and_end_key.first.Encode(), ikey_and_end_key.second);
 
+    Status s = tombstone_validator->Add(ikey_and_end_key.first.Encode(), ikey_and_end_key.second);
+    if(!s.ok()){
+      return s;
+    }
```

Comment:
I think the API doesn't require calls to `DeleteRange()` to follow start key order, unfortunately. Could we drop `tombstone_validator` from the PR for now and leave a TODO instead?

Reply:
Cool, I will drop the `tombstone_validator`. What will the TODO be about? Enforcing the order in the future ?

Reply:
Also, I ran into code format issues when submitting my PR. Is there any documentation about the tools available to format my code?